dev-util/pjdfstest/pjdfstest-0.1.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Exercise POSIX system calls"
+DESCRIPTION="pjdfstest is a test suite that helps exercise POSIX system calls."
+HOMEPAGE="http://people.freebsd.org/~pjd/pjdfstest"
+COPYRIGHT="2006-2012 Pawel Jakub Dawidek
+	Alan Somers
+	Ngie Cooper"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/pjd/pjdfstest/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="4701236d82bc13e0f0296bf920bc9b8433d519b2a862a5d8565a697ec3885954"
+PATCHES="pjdfstest-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	pjdfstest = $portVersion
+	cmd:pjdfstest = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	autoreconf -ifs
+	LDFLAGS="-lnetwork" runConfigure ./configure
```

Comment:
configure is created in the build, there is no mention of "socket, bind" or anything usefull atm in there ...